tools/wtdbg/.shed.yml

```diff
@@ -0,0 +1,11 @@
+name: wtdbg
+owner: bgruening
+description: WTDBG is a fuzzy Bruijn graph (FBG) approach to long noisy reads assembly.
+homepage_url: https://github.com/ruanjue/wtdbg-1.2.8
+long_description: |
```

Comment:
If you have multiple tools, or planning to have them, make a `suite` out of it.

Reply:
There won't be more tools. Would that be ok to keep it like this?